src/query/functions/linear/absent.go

```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/m3db/m3/src/query/executor/transform"
 )
 
+// FIXME: This is incorrect functionality.
+//' This should be an aggregation function that works in a step-wise fashion.
```

Comment:
Can we create an issue for this?

Reply:
Sure; actually did the impl for this during the PR but didn't want to pollute this with unrelated stuff even further haha; maybe I'll just jam it in here too, it's actually tiny

Reply:
Ah damn actually a bit trickier than what I had since it's not quite a "standard" aggregation (i.e. it creates its own series if one hasn't been passed in); will add an issue for now